ui/tailwind.config.cjs

```diff
@@ -1,6 +1,50 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  safelist: [
```

Comment:
There has to be a better way to get the Tailwind colors to work with Storybook, but I've changed the storybook preview imports, adjusted postCSS configs, and have not figured it out yet